e2e/lifecycle_test.go

```diff
@@ -15,6 +15,7 @@ import (
 	"github.com/prometheus/common/expfmt"
 	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
```

Comment:
Please check the status during initialization at `lifecycle_test.go`

Reply:
I changed the code to check `StatefulSetReady` like the code in upgrade_test.go.

Reply:
Has `lifecycle_test.go` been changed?

Reply:
I have updated lifecycle_test.go to check the status during initialization.
but the diff does not appears this PR. I think it's a bug in GitHub.
I changed in this commit.
https://github.com/cybozu-go/moco/pull/546/commits/bff9639abe08ce005c9904ef1f67e2d96fab58e8

Reply:
I have updated lifecycle_test.go to check the status during initialization.
but the diff does not appears this PR. I think it's a bug in GitHub.
I changed in this commit.
bff9639

Reply:
I can see it now!

Thank you update the test.
Please check the ReconcileSuccess Status just like `upgrade_test.go`.